examples/bootloader/.gitignore

```diff
@@ -0,0 +1 @@
+/keys.c
```

Comment:
Try $(BINDIR). If you create stuff in the application's directory, it breaks when building for two boards in parallel. (CI does that).